### PR TITLE
Add a Museums table

### PR DESCRIPTION
**Pull request by colagrosso**
Hi, @jobcurtis, this is the PR that I mentioned in #253 that includes some CSS changes.

The motivation from the PR came from a [previous discussion](https://github.com/standardebooks/web/issues/234#issuecomment-1587673991) about adding a `Museums` table and doing a simple check on the museum domain to make sure it's on the approved list.  Seemed useful, so I did it. 

I took the list of approved museums from [this list](https://standardebooks.org/manual/1.7.1/10-art-and-images#10.3.3.7.4) in SEMoS.

### CODE REVIEW

File: templates/ArtworkDetail.php
Comment (jobcurtis) [inline templates/ArtworkDetail.php:60]: Missing a closing `</p>` on this line

File: lib/Museum.php
Comment (jobcurtis) [inline lib/Museum.php:24]: Minor suggestion: you could do the search in the database rather than fetching all possibilities and searching them in PHP. It's a very small set though so really comes down to personal preference.

```suggestion
		$result = Db::Query('
			SELECT *
			FROM Museums
			WHERE ? LIKE CONCAT("%", Domain, "%")
			LIMIT 1;
		', [$url], 'Museum');

		return $result[0] ?? null;
```

**Review by jobcurtis — APPROVED** [review]
Looks good, approved pending merge of #253 & that one missing `</p>`.

File: templates/ArtworkDetail.php
Comment (colagrosso) [reply templates/ArtworkDetail.php]: Great catch, thanks. Fixed.

File: lib/Museum.php
Comment (colagrosso) [reply lib/Museum.php]: I like it. Updated. Thanks for proposing another way to do it.

My old `GetAll()` approach may have been better suited to using the APCu (or even just relying on the DB query cache), but this is less code.

**Comment by colagrosso** [comment]
Job, this PR depends on the CSS in #253 , so I'll wait for your approval on that PR before submitting both.

**Pull request merged**
